LocalSettings.php

```diff
@@ -1558,6 +1569,10 @@
 			'HP' => NS_PROJECT,
 			'HP_talk' => NS_PROJECT_TALK,
 		),
+		'+studynotekrwiki' => array(
+			'KSN' => NS_KOREAN_STUDY_NOTE,
```

Comment:
NS_STUDY_NOTE and NS_STUDY_NOTE_TALK are defined, but NS_KOREAN_STUDY_NOTE and NS_KOREAN_STUDY_NOTE_TALK aren't